### ISSUE TITLE
Script fails at instalation

### ISSUE THREAD

**Issue by ocenteno** [mallockey/Install-Office365Suite#2]
The script downloads the deployment tool correctly but then the installation stops and the output message is:

VERBOSE: received 3017488-byte response of content type application/octet-stream
VERBOSE: Running the Office Deployment Tool...
VERBOSE: Downloading and installing Office 365
WARNING: Office 365 was not detected after the install ran

Command run: 
.\Install-Office365Suite.ps1 -OfficeEdition O365BusinessRetail

**Comment by mallockey** [comment]
Hi @ocenteno , I tried manually running the command on my computer and it worked fine:

![Capture](https://user-images.githubusercontent.com/25836748/99561736-476b1c80-2995-11eb-8d53-5dac030ed09a.PNG)

Are there any other versions of Office installed on the machine you are trying on? I noticed quite a few logs generated in C:\Windows\Temp in the format of %COMPUTERNAME%-20201118-1151.txt maybe there is a hint in there as it why it failed.

Follow up question(s), does the black command prompt open after it downloads the Office Deployment tool? If so does it just quickly flash then close?
